backend/src/main/java/com/google/blackswan/mock/SimpleAnomalyGenerator.java

```diff
@@ -104,7 +104,10 @@ private void parseCSV() {
     scanner.close();
   }
 
-  /** Each row of csv has following format: { yyyy-mm-dd, popularity }. */
+  /** 
+   * Each row of csv has following format: { yyyy-mm-dd, popularity }. 
+   * ParseException takes 2 parameters: message, and index of string that failed to be parses.
```

Comment:
vvvvvvv nit: assuming message is the error message, add in "error" before message
parsed* instead of parses

Reply:
fixed it